changelogs/fragments/741-scaleway-database-backup-creation.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - scaleway_database_backup - new module for Scaleway provider Managed database backups (https://developers.scaleway.com/en/products/rdb/api/#backups-d14bc7).
```

Comment:
You can remove the changelog fragment, new modules are automatically announced in the changelog.